heareval/tasks/pipeline.py

```diff
@@ -300,58 +340,20 @@ def split_train_test_val(self, metadata: pd.DataFrame):
         metadata.loc[metadata["split_key"].isin(test_split_keys), "split"] = "test"
         return metadata
 
+    def get_requires_metadata_check(self, requires_key: str) -> pd.DataFrame:
+        df = self.get_requires_metadata(requires_key)
+        assert "relpath" in df.columns
+        assert "split" in df.columns
+        assert "label" in df.columns
+        if self.task_config["embedding_type"] == "event":
+            assert "start" in df.columns
+            assert "end" in df.columns
+        return df
+
     def run(self):
         # Process metadata gets all metadata to be used for the task
         metadata = self.get_all_metadata()
 
-        # Deterministically shuffle the metadata
```

Comment:
Looks like this all got moved out into a separate function `postprocess_all_metadata`, but that function is never called

Reply:
oups caught that in the upcoming PR #23